tests/python/unittest/test_meta_schedule_schedule_rule_multi_level_tiling.py

```diff
@@ -560,7 +563,12 @@ def test_multi_level_tiling_dense_dpa4():
         )
     ]
 
-    check_trace(spaces, expected)
+    _test_multi_level_tiling_dense_dp4a(m, n, k, "int8", "int32", expected)
+
+
+def test_multi_level_tiling_dense_dp4a_non_tensorizable():
+    _test_multi_level_tiling_dense_dp4a(128, 128, 128, "float32", "float32", [""])
+    _test_multi_level_tiling_dense_dp4a(127, 127, 127, "int32", "int32", [""])
```

Comment:
what about `_test_multi_level_tiling_dense_dp4a(127, 127, 127, "int8", "int32", [""])`? what happen if the dtype matches with the intrin and the only mismatch is the shape?

Reply:
Sorry this was a typo, I wanted to have `"int8", "int32"`. To my surprise, I got `Check failed: (tiled_loop_rv.defined()) is false` error. 

Fixing.

Reply:
Fixed